tests/test_uncrustify/test.py

```diff
@@ -0,0 +1,211 @@
+# Class encapsulating a unit test.
+#
+# * @author  Ben Gardner        October 2009
+# * @author  Guy Maurel         October 2015
+# * @author  Matthew Woehlke    June 2018
+#
+
+import filecmp
+import os
+import subprocess
+import sys
+
+from .ansicolor import printc
+from .config import (config, test_dir, FAIL_ATTRS,
+                     MISMATCH_ATTRS, UNSTABLE_ATTRS)
+from .failure import (ExecutionFailure, MissingFailure,
+                      MismatchFailure, UnstableFailure)
+
+
+# =============================================================================
+class Test(object):
+    pass_config = ['test_config', 'test_rerun_config']
+    pass_input = ['test_input', 'test_rerun_expected']
```

Comment:
maybe I got it wrong, but shouldn't it be like this `pass_input = ['test_input', 'test_expected']` ?

Reply:
Well... that would be the discussion (from https://github.com/uncrustify/uncrustify/pull/1799#discussion_r195225473) of unclear resolution. (Note: this behavior matches what is in master (with CTest).)

Reply:
Maybe we should leave a note in the code.